MIT.md

```diff
@@ -1,10 +1,8 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2022, Okolo Oshokenoya
```

Comment:
- [ ] [OPTIONAL] Nice addition here, but am afraid it's not necessary at this stage, keep up the good work.

Reply:
I'll be sure to take note of this 

> * [ ]  [OPTIONAL] Nice addition here, but am afraid it's not necessary at this stage, keep up the good work.

Alright. Duly noted